ambry-frontend/src/main/java/com.github.ambry.frontend/FrontendMetrics.java

```diff
@@ -294,30 +327,16 @@ public FrontendMetrics(MetricRegistry metricRegistry) {
 
     // Latencies
     // AmbryBlobStorageService
-    // DELETE
-    deletePreProcessingTimeInMs =
-        metricRegistry.histogram(MetricRegistry.name(AmbryBlobStorageService.class, "DeletePreProcessingTimeInMs"));
-    // HEAD
-    headPreProcessingTimeInMs =
-        metricRegistry.histogram(MetricRegistry.name(AmbryBlobStorageService.class, "HeadPreProcessingTimeInMs"));
-    // GET
-    getPreProcessingTimeInMs =
-        metricRegistry.histogram(MetricRegistry.name(AmbryBlobStorageService.class, "GetPreProcessingTimeInMs"));
     // POST
     blobPropsBuildTimeInMs =
         metricRegistry.histogram(MetricRegistry.name(AmbryBlobStorageService.class, "BlobPropsBuildTimeInMs"));
-    postPreProcessingTimeInMs =
-        metricRegistry.histogram(MetricRegistry.name(AmbryBlobStorageService.class, "PostPreProcessingTimeInMs"));
     // OPTIONS
     optionsPreProcessingTimeInMs =
```

Comment:
I don't understand the ordering of the metrics? Looks like  `optionsPreProcessingTimeInMs ` can move up to 304.

Reply:
I want to isolate the metrics that are AsyncOperationTracker.Metrics instances from the ones that are just histograms. I did not refactor options to use async security checks yet since it currently does everything in one method.